compact_proof/groth16/p0tionConfig.json

```diff
@@ -1,30 +1,30 @@
 {
-  "title": "RISC Zero Groth16 Verifier",
-  "description": "A Ceremony for the Groth 16 circuit used in RISC Zero's STARK-to-SNARK code",
-  "startDate": "2023-12-04T23:26:22",
-  "endDate": "2024-01-03T23:26:22",
+  "title": "RISC Zero STARK-to-SNARK Prover",
+  "description": "A circuit which verifies RISC Zero recursive STARKs, converting them into Groth16 SNARKs",
+  "startDate": "2024-04-03T15:00:00",
+  "endDate": "2024-05-02T03:00:00",
   "timeoutMechanismType": "FIXED",
   "penalty": 10,
   "circuits": [
     {
-      "description": "This circuit verifies the validity of a RISC Zero recursion receipt",
+      "description": "RISC Zero STARK-to-SNARK Prover",
       "compiler": {
         "version": "2.1.6",
         "commitHash": "57b18f68794189753964bfb6e18e64385fed9c2c"
       },
       "template": {
         "source": "https://github.com/risc0/risc0",
-        "commitHash": "41a0997a68ae288f5456238ca9ae9c2b68c37421",
+        "commitHash": "2c2d7187896ca1e4c94428be6fadf36240b1d7f0",
         "paramsConfiguration": []
       },
       "verification": {
         "cfOrVm": "CF"
       },
       "artifacts": {
-        "r1csStoragePath": "http://risc0-p0tion-ceremony.s3.us-east-2.amazonaws.com/verify.r1cs",
-        "wasmStoragePath": "http://risc0-p0tion-ceremony.s3.us-east-2.amazonaws.com/verify.wasm"
+        "r1csStoragePath": "http://risc0-artifacts.s3.us-east-2.amazonaws.com/stark_verify.r1cs",
+        "wasmStoragePath": "http://risc0-artifacts.s3.us-east-2.amazonaws.com/stark_verify.wasm"
```

Comment:
We might want to have this be versioned (perhaps via timestamp), just in case we need to turn another ceremony down the road.